### ISSUE TITLE
Default test files are generating repeating text / hallucinating 

### ISSUE THREAD

**Issue by greenman2** [sanchit-gandhi/whisper-jax#9]
I used the default settings on the Kaggle notebook.

[https://huggingface.co/datasets/sanchit-gandhi/whisper-jax-test-files
](https://huggingface.co/datasets/sanchit-gandhi/whisper-jax-test-files
)
![i000056](https://user-images.githubusercontent.com/5298065/233511605-381a24da-cd66-4e04-9bca-a9a5d4d8578a.jpg)

**Comment by sanchit-gandhi** [comment]
Looks like a hallucination (known issue of the Whisper model) - worth trying with timestamps

**Comment by sanchit-gandhi** [comment]
We could implement a logits processor that stops repetition of repeated tokens (similar to what we have in PyTorch: https://github.com/huggingface/transformers/blob/04ab5605fbb4ef207b10bf2772d88c53fc242e83/src/transformers/generation/logits_process.py#L474)

This might be quite difficult to do in a JAX compiled way though without returning the value of each token generated

**Comment by rpatel15-hue** [comment]
See this issue can be improved by reducing the --compression_ratio_threshold from default 2.4. https://github.com/openai/whisper/discussions/192
How can I modify this with this library?
